kmeleon.txt

```diff
@@ -1,35 +1,34 @@
 [Section]
 Name = K-Meleon
-Version = 75.1
+Version = 76.4.7
 License = GPL
-Description = An extremely fast, customizable and still lightweight web browser based on the Gecko layout engine.
+Description = An extremely fast, customizable and still lightweight web browser based on the Goanna layout engine.
 Category = 5
-URLSite = http://kmeleon.sourceforge.net
-URLDownload = http://download.sourceforge.net/project/kmeleon/k-meleon/75.1/K-Meleon75.1.exe
-SHA1 = a46ead69a6d18609b1f2e8ae60a3a5883e2e061e
-SizeBytes = 26570888
+URLSite = http://kmeleonbrowser.org/forum/read.php?19,154431
```

Comment:
I checked: Using https here would not work. Not even from Windows, so this is fine.